### DISCUSSION TITLE
Default tab in forms

### DISCUSSION THREAD

**Discussion by BTreeTech** [opengisch/QField#5417] [Q&A]
I have designed a form for data entry, it works through the data sequentially but the first tab visible is always the last tab used for the previous feature added. Is there a way to set the default tab visible at feature creation to be the first one not the last used?
I could hide the others until the first one is complete but being able to change the default would be nice.

**Comment by MidnightCoder-m** [comment]
Hi @BTreeTech, Thanks for reporting, please try the APKs from  [**this PR.**](https://github.com/opengisch/QField/pull/5436) I am closing this as I believe the issue is now fixed.
